src/components/DetailsModal/DetailsModal.jsx

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from "react";
+
+import './style.scss';
+
+class DetailsModal extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isModalShowing: false,
+    }
+    this.toggleModal = this.toggleModal.bind(this);
+  }
+
+  toggleModal () {
+    this.setState(state => ({
+      isModalShowing: !state.isModalShowing
+    }));
+  }
+
+  Modal() {
```

Comment:
Why is it in a separate function?

Reply:
Modal structure is gonna change I just pushed as it was yesterday. I need to research it more.